nuxeo-distribution/nuxeo-nxr-server/src/main/resources/templates/mongodb/nuxeo.defaults

```diff
@@ -26,7 +26,7 @@ nuxeo.dbs.cache.concurrencyLevel=10
 nuxeo.dbs.cache.ttl=10
 
 ## Startup checks configuration
-mongodb.check.classpath=nxserver/lib:${nuxeo.home}/nxserver/bundles/nuxeo-core-storage-mongodb-*.jar
+mongodb.check.classpath=nxserver/lib:${nuxeo.home}/nxserver/bundles/nuxeo-*-mongodb-*.jar
```

Comment:
I would prefer to be explicit. I don't want to catch any other future random Nuxeo JAR that have mongodb in its name, even if we don't have them today.

Reply:
`MongoDBChecker` is in `nuxeo-core-storage-mongodb`, so the jar is needed.